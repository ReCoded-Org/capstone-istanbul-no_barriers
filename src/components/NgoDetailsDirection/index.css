.google-map {
  height: 20rem;
  width: 80%;
  color: white;
  text-transform: uppercase;
  font-weight: bold;
}
