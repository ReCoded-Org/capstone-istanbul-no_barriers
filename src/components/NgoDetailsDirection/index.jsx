import React from "react";
import GoogleMapReact from "google-map-react";
import LocationPin from "../LocationPin";
import "./index.css";

const NgoDetailsDirection = ({ address }) => {
  const location = {
    address,
    lat: 37.42216,
    lng: -122.08427,
  };

  return (
    <>
      <h5>Directions</h5>
      <div className="google-map">
        <GoogleMapReact
          bootstrapURLKeys={{ key: "AIzaSyA_W5FFmcpoLooDBWggCdOaWdD20aacVXk" }}
          defaultCenter={location}
          defaultZoom={8}
        >
          <LocationPin
            lat={location.lat}
            lng={location.lng}
            text={location.address}
          />
        </GoogleMapReact>
      </div>
    </>
  );
};

export default NgoDetailsDirection;
