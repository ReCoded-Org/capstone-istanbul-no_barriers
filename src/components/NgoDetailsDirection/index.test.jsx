import React from "react";
import renderer from "react-test-renderer";
import NgoDetailsDirection from "./index";

it("NgoDetailsDirection renders correctly", () => {
  const tree = renderer
    .create(<NgoDetailsDirection address="Ankara" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
