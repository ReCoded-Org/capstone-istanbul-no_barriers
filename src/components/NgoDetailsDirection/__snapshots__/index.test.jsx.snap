// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NgoDetailsDirection renders correctly 1`] = `
Array [
  <h5>
    resources.ngoList.ngoDetails.directions
  </h5>,
  <div
    className="google-map"
  >
    <div
      onClick={[Function]}
      onMouseDownCapture={[Function]}
      onMouseMove={[Function]}
      style={
        Object {
          "height": "100%",
          "margin": 0,
          "padding": 0,
          "position": "relative",
          "width": "100%",
        }
      }
    >
      <div
        style={
          Object {
            "height": "100%",
            "left": 0,
            "margin": 0,
            "padding": 0,
            "position": "absolute",
            "top": 0,
            "width": "100%",
          }
        }
      />
      <div
        style={
          Object {
            "height": "50%",
            "left": "50%",
            "margin": 0,
            "padding": 0,
            "position": "absolute",
            "top": "50%",
            "width": "50%",
          }
        }
      >
        <div
          style={
            Object {
              "height": "100%",
              "left": 0,
              "margin": 0,
              "padding": 0,
              "position": "absolute",
              "top": 0,
              "width": "100%",
            }
          }
        >
          <div
            style={
              Object {
                "backgroundColor": "transparent",
                "height": 0,
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": 0,
              }
            }
          >
            <div
              className="pin"
            >
              <span
                className="iconify"
                data-icon="mdi:map-marker"
                data-inline="false"
              />
              <p
                className="pin-address"
              >
                Ankara
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
